experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -791,6 +842,16 @@ export function intoDelta(
 		});
 		rootDelta.build = builds;
 	}
+	if (change.destroys && change.destroys.size > 0) {
+		const destroys: Delta.DetachedNodeDestruction[] = [];
+		forEachInNestedMap(change.destroys, (_, major, minor) => {
+			destroys.push({
+				id: makeDetachedNodeId(major ?? revision, minor),
+				count: 1,
+			});
+		});
+		//  rootDelta.destroys = destroys;
```

Comment:
Why is this commented out?

Reply:
That was waiting on the previous PR to merge in.